app/src/main/java/comp3350/mealbuddy/objects/Food.java

```diff
@@ -0,0 +1,49 @@
+package comp3350.mealbuddy.objects;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Dictionary;
+import java.util.EnumMap;
+import java.util.Enumeration;
+import java.util.Hashtable;
+
+public class Food {
+
+
+    private String name;
+
+    private ArrayList<String>labels;
+
+    private EnumMap macroDictionary;
+
+    private EnumMap microDictionary;
+
+    public Food (String name) {
+        this.name = name;
+        labels = new ArrayList<String>();
+        macroDictionary = Macros.makeMacros();
+        microDictionary = Micros.makeMicros();
```

Comment:
same as above, this doesn't make sense. it assumes that a piece of food has EVERY nutrient in it.